src/ArmTemplates/Common/API/Clients/Service/ServiceApisApiClient.cs

```diff
@@ -0,0 +1,46 @@
+﻿// --------------------------------------------------------------------------
+//  <copyright file="ManagementInstanceApiClient.cs" company="Microsoft">
+//      Copyright (c) Microsoft Corporation. All rights reserved.
+//  </copyright>
+// --------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.API.Clients.Abstractions;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.API.Clients.Service.Responses;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Constants;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Templates.Service;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Extractor.Models;
+
+namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.API.Clients.Service
+{
+    public class ServiceApisApiClient : ApiClientBase, IServiceApisApiClient
```

Comment:
I can try to explain my intention in naming:
- class that is sending request is named like `<what-object-you-are-processing>ApiClient`, because it is sending api requests.
- I wanted this class to get APIM instance APIs. So object I am processing is `ServiceApis` (aka APIM APIs). I didn't want to write ApimApis because there will be quite a lot of abbreviations and I prefer full-naming for everything, so I decided to name APIM as a Service.
-> And so we have a `ServiceApis ApiClient` naming.

Please, provide your thoughts on this naming!

Reply:
If you mean a logical API, then just call it Api, no? Or LogicalApi? No need to include service then because it feels like you are talking about the APIM management API.

Reply:
> there will be quite a lot of abbreviations and I prefer full-naming for everything, so I decided to name APIM as a Service.

Agreed! I try to avoid using abbreviations as well

Reply:
I would just call it an `ApisClient`.
Regarding `ApiClient` - Does it add any value for the dev to know that it calls an API instead of using an SDK or something? Do we have other ApisClient implementations that we need to distinguish between the API client and others?
`ServiceApis` - Same thing here. Does it add any value to have the `service` in the name? I think it's kind of obvious it's getting the Apis from an APIM instance/service. It's the purpose of this tool.